src/pepr/operator/index.ts

```diff
@@ -83,3 +92,13 @@ When(UDSPackage)
     log.info("Identity and Authorization layer removed, operator will NOT handle SSO.");
     UDSConfig.isIdentityDeployed = false;
   });
+
+// Watch for changes to the API server EndpointSlice and update the API server CIDR
+if (!UDSConfig.kubeNodeCidrs) {
+  When(a.Node).IsCreatedOrUpdated().Reconcile(updateKubeNodesFromCreateUpdate);
+}
+
+// Watch for changes to the API server EndpointSlice and update the API server CIDR
```

Comment:
These comments don't match the functionality here (should be `Watch for changes to nodes ...`).

Reply:
```suggestion
// Watch for changes to the Nodes and update the Node CIDR list
if (!UDSConfig.kubeNodeCidrs) {
  When(a.Node).IsCreatedOrUpdated().Reconcile(updateKubeNodesFromCreateUpdate);
}

// Watch for Node deletions and update the Node CIDR list
```